browser/ui/webui/brave_vpn/brave_vpn_localized_string_provider.cc

```diff
@@ -50,10 +50,14 @@ void AddLocalizedStrings(content::WebUIDataSource* html_source) {
       {"braveVpnLoading", IDS_BRAVE_VPN_LOADING},
       {"braveVpnPurchaseFailed", IDS_BRAVE_VPN_PURCHASE_FAILED},
       {"braveVpnSelectYourServer", IDS_BRAVE_VPN_SELECT_YOUR_SERVER},
-      {"braveVpnServerSelectionCountryInfo",
-       IDS_BRAVE_VPN_SERVER_SELECTION_COUNTRY_INFO},
-      {"braveVpnServerSelectionCityInfo",
-       IDS_BRAVE_VPN_SERVER_SELECTION_CITY_INFO},
+      {"braveVpnServerSelectionSingleCity",
```

Comment:
I think you should use a plural string helper here instead as some languages have different terms for different numbers of things - see
- `IDS_BRAVE_NEWS_SOURCE_COUNT` for an example (and `brave/components/brave_new_tab_ui/components/default/braveNews/customize/SourcesList.tsx`)